src/psyclone/tests/test_files/dynamo0p3/infrastructure/argument_mod.F90

```diff
@@ -161,7 +161,8 @@ module argument_mod
   integer, public, parameter :: X1D        = 2
   integer, public, parameter :: Y1D        = 3
   integer, public, parameter :: CROSS      = 4
-  integer, public, parameter :: CROSS2D    = 5
+  integer, public, parameter :: REGION     = 5
```

Comment:
Please update the copyright date for the modifications to this file to "2020-2021"

Reply:
Done, see commit 77b51c2ebc81fb1860b1b0f15c9b48b73e20c70b